django/db/backends/oracle/creation.py

```diff
@@ -77,9 +77,6 @@ def _create_test_db(self, verbosity=1, autoclobber=False, keepdb=False):
             try:
                 self._create_test_user(cursor, parameters, verbosity, keepdb)
             except Exception as e:
-                # If we want to keep the db, then we want to also keep the user.
-                if keepdb:
-                    return
```

Comment:
IMO the only situation in which `_create_test_user` can raise exception that we want to omit (when `--keepdb` flag is on) is already checked in this [line](https://github.com/django/django/blob/master/django/db/backends/oracle/creation.py#L225) (`ORA-01920: user name 'string' conflicts with another user or role name`).